samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<p>Why not use Path to verify/make the directory?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>